apps/architecture-demos/src/app/data-access/github/+state/selectors.ts

```diff
@@ -1,7 +1,7 @@
 import { createSelector } from '@ngrx/store';
 import { GitHubFeatureState, GitHubState } from './reducer';
 
-export const selectGitHub = (globalState: GitHubFeatureState) => {
+export const selectGitHub = (globalState: any) => {
```

Comment:
I thought ngrx is not part of the demos anymore?

Reply:
Yes, not in that state-demo app.

Here I want to demo some architecture and need different global state managements